pkgs/by-name/bo/borealis-cursors/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation {
```

Comment:
You should use `stdenvNoCC.mkDerivation(finalAttrs: { ` here, so that you can reuse the `version` attribute down in the `src` attribute.